backend/maelstro/main.py

```diff
@@ -0,0 +1,91 @@
+"""
+Main backend app setup
+"""
+
+from typing import Annotated
+from fastapi import FastAPI, Request, Response, Header
+
+
+app = FastAPI(root_path="/maelstrob")
+
+app.state.health_countdown = 5
+
+
+@app.head("/")
+@app.get("/")
+def root_page():
+    """
+    Hello world dummy route
+    """
+    return {"Hello": "World"}
+
+
+@app.get("/user")
+def user_page(
+    sec_username: Annotated[str | None, Header(include_in_schema=False)] = None,
+    sec_org: Annotated[str | None, Header(include_in_schema=False)] = None,
+    sec_roles: Annotated[str | None, Header(include_in_schema=False)] = None,
+    sec_external_authentication: Annotated[
+        str | None, Header(include_in_schema=False)
+    ] = None,
+    sec_proxy: Annotated[str | None, Header(include_in_schema=False)] = None,
+    sec_orgname: Annotated[str | None, Header(include_in_schema=False)] = None,
+):
+    """
+    Display user information provided by gateway
+    """
+    return {
+        "username": sec_username,
+        "org": sec_org,
+        "roles": sec_roles,
+        "external-authentication": sec_external_authentication,
+        "proxy": sec_proxy,
+        "orgname": sec_orgname,
+    }
+
+
+@app.head("/debug")
+@app.get("/debug")
+@app.put("/debug")
+@app.post("/debug")
+@app.delete("/debug")
+@app.options("/debug")
+@app.patch("/debug")
+def debug_page(request: Request):
```

Comment:
May desserve a:

```python
#TODO: deactivate for prod
```

Could be done later using an environment variable.

Reply:
grrr, pylint does not like TODOs ;-)

fixed